common/lib/xmodule/xmodule/course_module.py

```diff
@@ -294,6 +294,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_("Enter the url of the official Facebook page of the course."),
+        default="",
+        display_name=_("Course Facebook Page"),
+        scope=Scope.settings
+    )
```

Comment:
@nasthagiri , is it a page or a group? is it only for use with Facebook? I think I need a few minutes with @lou-wang to understand this setting and its purpose before I can comment on the label or its help.